test/test_nn.py

```diff
@@ -6058,6 +6058,168 @@ def test_shape(N, C, ID, IH, IW, D, H, W, mode, padding_mode, align_corners):
                     for input_requires_grad in [False, True]:
                         test(N, C, D, H, W, mode, padding_mode, align_corners, input_requires_grad)
 
+    def test_grid_sample_nearest_neighbor_rounding_mode_consistency(self):
+
+        device_list = ['cpu']
+        if TEST_CUDA:
```

Comment:
I believe this decoration is only used for testing the implementations that's only available for CUDA. However, my unit test needs to be at least tested for CPU. If there is CUDA availability, the CUDA implementations should also be tested. Therefore, I think I cannot use this decorator. Otherwise, if the user does not have CUDA in their platform, this unit test will not run.